config/locales/nl.yml

```diff
@@ -222,7 +222,7 @@ nl:
       comments: 'Commentaar:'
   meetings:
     singular: Samenkomst
-    edit_meeting: 'Bewerk Samenkomst'
+    edit_meeting: 'Bewerk samenkomst'
```

Comment:
I'd really like to call it Samenkomst, line 253 also is capitalized Geheim Delen, etc, they are names of sections and the capital marks them as such. That's no official spelling rule in Dutch but it is consistent.

Reply:
I doubted about a lot of these words with capital letters. I feel it is something from the programming world to capitalize every first letter but I can imagine it seems quite odd for Dutch people. But it is true that we need to be consistent in this.